src/pages/Home/index.js

```diff
@@ -39,6 +42,23 @@ class Home extends React.Component {
                 />
               ))}
           </Box>
+          <Box direction="column">
```

Comment:
You don't need to say direction="column" since it's the default

Reply:
ah ok, got it.